Sources/SwiftSyntax/generated/syntaxNodes/SyntaxStmtNodes.swift

```diff
@@ -1915,6 +1915,128 @@ public struct ReturnStmtSyntax: StmtSyntaxProtocol, SyntaxHashable {
   }
 }
 
+// MARK: - ThenStmtSyntax
+
+/// ### Children
+/// 
+///  - `thenKeyword`: `'then'`
+///  - `expression`: ``ExprSyntax``
+public struct ThenStmtSyntax: StmtSyntaxProtocol, SyntaxHashable {
```

Comment:
@ahoppen Any thoughts on whether the syntax nodes for experimental features should be `public`? Makes sense if the experimental feature itself is public, though means we'll have a lot more deprecated API surface area if we ever decide to remove an experimental feature

Reply:
How about hiding such an API behind `@_spi`?

Reply:
Yes, that's the alternative I'm thinking of :)

Reply:
I don’t think we can hide it behind `@_spi`. We would need to make the `thenStmt` case in `SyntaxEnum` SPI and I don’t think you can mark individual cases as SPI (how would you switch exhaustively over such an enum).

So, I think we just need to make them public. If we decide to rip them out before the 510 release, we don’t need to add deprecations either since nobody should be using them.

Reply:
FWIW I'm pretty sure we do support `@_spi` for enum cases, clients switching over resilient enums already have to account for unknown cases with `@unknown default`

Reply:
Oh, it indeed works, even if you don’t enable library evolution. In that case putting the experimental nodes behind `@_spi(ExperimentalLanguageFeatures)` would be a good idea.

Reply:
Updated to mark as SPI